composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "meanbee/magento-cmspagerevisions",
+    "description": "Maintain a history of CMS pages and static blocks in Magento 1 with this extension.",
+    "homepage": "https://www.meanbee.com/magento-extensions/cms-revisions.html",
+    "type": "magento-module",
+    "version": "1.1.0",
+    "authors": [
+        {
+            "name": "Meanbee",
+            "email": "hello@meanbee.com"
+        }
+    ],
+	"require":{
```

Comment:
Looks good, could you just update the indentation with the require lines and I'll get merged in. Thanks.